applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element_discontinuous.cpp

```diff
@@ -131,8 +131,8 @@ void EmbeddedFluidElementDiscontinuous<TBaseElement>::CalculateLocalSystem(
         }
 
         // Recall to swap the boundary term signs because of the interface equilibrium (Neumann) imposition
```

Comment:
Is this comment still relevant?

Reply:
No. Indeed, it is misleading. Actually, we can even remove the `aux_LHS` and `aux_RHS` arrays.